python/ola/RDMTest.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
+# RDMTest.py
+# Copyright (C) 2019 Bruce Lowekamp
+
+import binascii
+import os
+import socket
+# import timeout_decorator
+import unittest
+from ola import PidStore
+from ola.ClientWrapper import ClientWrapper
+from ola.RDMAPI import RDMAPI
+from ola.UID import UID
+
+
+"""Test cases for RDM device commands."""
+
+__author__ = 'bruce@lowekamp.net (Bruce Lowekamp)'
+
+global pidStorePath
+
+
+class RDMTest(unittest.TestCase):
+  # @timeout_decorator.timeout(2)
+  def testGetWithResponse(self):
+    """uses client to send an RDM get with mocked olad.
+    Regression test that confirms sent message is correct and
+    sends fixed response message."""
+    sockets = socket.socketpair()
+    wrapper = ClientWrapper(sockets[0])
+    pid_store = PidStore.GetStore(pidStorePath)
+    client = wrapper.Client()
+    rdm_api = RDMAPI(client, pid_store)
+
+    class results:
+      gotrequest = False
+      gotresponse = False
```

Comment:
Perhaps switch to this to match the other code:
```suggestion
      got_request = False
      got_response = False
```

Then equivalent matching changes throughout.

Reply:
I've written to so many coding standards over the years that I tend to be entirely inconsistent in naming variables unless I think about it...